VEX/vex.json

```diff
@@ -83,7 +83,7 @@
       "analysis": {
         "state": "not_affected",
         "justification": "code_not_reachable",
-        "response": ["will_not_fix", "update"],
+        "response": "will_not_fix",
```

Comment:
:x: per aplied JSON schema `response` is to be a list of known values. a LIST.
see also https://cyclonedx.org/docs/1.4/json/#vulnerabilities_items_analysis_response